CMakeLists.txt

```diff
@@ -57,6 +57,7 @@ option(PAASS_BUILD_ANALYSIS "Build analysis related programs" ON)
 option(PAASS_BUILD_SETUP "Include the older setup programs in installation" OFF)
 option(PAASS_BUILD_SHARED_LIBS "Install only scan libraries" ON)
 option(PAASS_BUILD_TESTS "Builds programs designed to test the package. Including UnitTest++ test." OFF)
+option(PAASS_BUILD_XIA_INTERFACE "" ON)
```

Comment:
What exactly is this doing? I don't see any use of having this flag in the CMakeLists.txt yet.